acquire/utils.py

```diff
@@ -323,6 +328,11 @@ def check_and_set_acquire_args(
     if not args.children and args.skip_parent:
         raise ValueError("--skip-parent can only be set with --children")
 
+    if args.output_type == "zip" and args.compress_method not in ZIP_COMPRESSION_METHODS:
+        raise ValueError(f"Invalid compression method for zip: {ZIP_COMPRESSION_METHODS.keys()}")
```

Comment:
```suggestion
        raise ValueError(f"Invalid compression method for zip, allowed are: {ZIP_COMPRESSION_METHODS.keys()}")
```

Otherwise it reads as if these methods are invalid